src/common/legacy_config_opts.h

```diff
@@ -394,6 +394,7 @@ OPTION(client_metadata, OPT_STR)
 OPTION(client_acl_type, OPT_STR)
 OPTION(client_permissions, OPT_BOOL)
 OPTION(client_dirsize_rbytes, OPT_BOOL)
+OPTION(client_deleg_timeout, OPT_DOUBLE)
```

Comment:
We don't add to this file anymore. Please add to options.cc instead. See 06c94de584e6cd7d347bcdfb79d9fef4fed0d277 for a recent example.